CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.0 - Sept 4, 2023
```

Comment:
is this actually a major version? I'd consider it a minor since it's expected for everyone to have moved on from this version

Reply:
Hmm you're right this would be a minor